test/services/sync/strategies/CSVSyncer/CSVSyncer.integration.test.js

```diff
@@ -1142,4 +1142,69 @@ describe('CSVSyncer Integration', () => {
 			expect(class1b.year.toString()).to.equal(scenario1.school.currentYear._id.toString());
 		});
 	});
+
+	describe('Scenario 14 - Attribute aliases', () => {
+		let scenarioParams;
+		let scenarioData;
+
+		const SCHOOL_ID = testObjects.options.schoolId;
+		const TEACHER_EMAILS = [
+			'a@b.de',
+			'b@c.de',
+			'c@d.de',
+		];
+
+		before(async () => {
+			const user = await createUser({ roles: 'administrator', schoolId: SCHOOL_ID });
+			scenarioParams = await generateRequestParamsFromUser(user);
+			scenarioParams.query = {
+				target: 'csv',
+				school: SCHOOL_ID,
+				role: 'teacher',
+			};
+			scenarioData = {
+				data: 'affix,first,middle,last,eMail\n'
+					+ `Dr.,Peter,F.,Pan,${TEACHER_EMAILS[0]}\n`
+					+ `Mr.,Peter,,Lustig,${TEACHER_EMAILS[1]}\n`
+					+ `HM,Test,T.,Testington,${TEACHER_EMAILS[2]}\n`,
+			};
+		});
+
+		after(async () => {
+			await testObjects.cleanup();
+			await deleteUser(TEACHER_EMAILS[0]);
+			await deleteUser(TEACHER_EMAILS[1]);
+			await deleteUser(TEACHER_EMAILS[2]);
+		});
+
+		it('should be accepted for execution', () => {
+			expect(CSVSyncer.params(scenarioParams, scenarioData)).to.not.equal(false);
```

Comment:
what does .params do?

Reply:
It validates the params as defined in the Syncer interface (in this case `role`, (CSV-)`data`, `sendEmails?`)